src/component-library/utils/prop-types.ts

```diff
@@ -6,12 +6,16 @@ const status = tuple('error', 'warning', 'success');
 
 const sizes = tuple('small', 'medium');
 
+export const colors = tuple('primary', 'secondary', 'tertiary');
```

Comment:
Should have picked this up before so not critical, but what's the benefit of using a tuple here?

Reply:
@tomjeatt it's just syntax, but simply using arrays should do the job as well.

Reply:
I can change this to arrays.